docs/core/integration-quality-scale/rules/iqs029.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "IQS029 - Strict typing"
+---
+
+IQS029 is as follows:
+
+Strict typing
+
+## Reasoning
+
+Python is a dynamically typed language, which can be the source of many bugs.
+By using type hints, you can catch bugs early and avoid introducing bugs.
+
+Type hints are checked by mypy, a static type checker for Python.
+Because the way typing in Python works, this still isn't waterproof.
+To improve on this, we recommend fully typing your library and making your library PEP-561 compliant.
+This means that you need to add a `py.typed` file to your library, which tells mypy that your library is fully typed, after which it can read the type hints from your library.
```

Comment:
Waterproof is a Dutch literal translation :)

```suggestion
Type hints are checked by mypy, a static type checker for Python.
Because of the way typing in Python works, this still isn't watertight.
To improve on this, we recommend fully typing your library and making your library PEP-561 compliant.
This means that you need to add a `py.typed` file to your library. This file tells mypy that your library is fully typed, after which it can read the type hints from your library.
```